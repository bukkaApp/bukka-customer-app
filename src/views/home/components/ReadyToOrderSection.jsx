import React from 'react';

import SearchLocation from '../../../components/places-suggest/SearchLocation';

import './readyToOrderSection.scss';

const ReadyToOrderHeader = () => (
  <div className="header-section">
    <h2 className="header text-center"> Ready to order?</h2>
  </div>
);

const ReadyToOrderSection = () => (
  <div className="ready-to-order-section">
    <div className="container">
      <ReadyToOrderHeader />
      <div className="max-search-location mx-auto">
        <SearchLocation />
      </div>
    </div>
  </div>
);

export default ReadyToOrderSection;

ReadyToOrderSection.propTypes = {};
