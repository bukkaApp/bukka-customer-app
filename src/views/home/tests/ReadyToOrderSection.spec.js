import React from 'react';
import ReadyToOrderSection from 'src/components/ReadyToOrderSection';

describe('Ready to order section component', () => {
  const wrrapper = shallow(<ReadyToOrderSection />);

  it('renders properly', () => {
    expect(wrrapper).toMatchSnapshot();
  });
});
