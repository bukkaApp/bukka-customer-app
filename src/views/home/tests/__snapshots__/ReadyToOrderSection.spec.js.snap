// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ready to order section component renders properly 1`] = `ShallowWrapper {}`;
